src/index.js

```diff
@@ -1,15 +1,12 @@
 import './style.css';
 
+import { saveTasks, savedTasks } from './modules/edit.js';
+
 const taskInput = document.getElementById('input');
 const addButton = document.getElementById('button');
 const todoList = document.getElementById('todo-list');
-const completeButton = document.querySelector('.complete');
-
-const savedTasks = JSON.parse(localStorage.getItem('tasks')) || [];
 
-function saveTasks() {
-  localStorage.setItem('tasks', JSON.stringify(savedTasks));
-}
+// const completeButton = document.querySelector('.complete');
 
 function removeTask(li) {
   const index = Array.prototype.indexOf.call(todoList.children, li);
```

Comment:
Great job @SAZZAD1Q2 🤝

- [ ] [Optional] Kindly ensure the todo indexes update every time you delete an item from your todo list.